docs/pages/Command-Line.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: default
+title: Command-Line
+permalink: /command-line/
+---
+
+- _TODO: update "Command-Line" document [#868](https://github.com/ChainSafe/gossamer/issues/868)_
+
+---
+
+## Gossamer Command
+
+The `gossamer` command is the root command for the `gossamer` package (`cmd/gossamer`). The root command starts the node (and initializes the node if the node has not already been initialized). 
+
+The `gossamer` command accepts the following ***local flags*** and ***global flags***:
+
+```
+--verbosity value  Supports levels crit (silent) to trce (trace) (default: "info")
+--name value       Node implementation name
+--chain value       Node implementation id used to load default node configuration
+--config value     TOML configuration file
+--datadir value    Data directory for the node
+--key value        Specify a test keyring account to use: eg --key=alice
+--unlock value     Unlock an account. eg. --unlock=0,2 to unlock accounts 0 and 2. Can be used with --password=[password] to avoid prompt. For multiple passwords, do --password=password1,password2
+--port value       Set network listening port (default: 0)
+--bootnodes value  Comma separated enode URLs for network discovery bootstrap
+--protocol value   Set protocol id
+--roles value      Roles of the gossamer node
+--nobootstrap      Disables network bootstrapping (mdns still enabled)
+--nomdns           Disables network mdns discovery
+--rpc              Enable the HTTP-RPC server
+--rpchost value    HTTP-RPC server listening hostname
+--rpcport value    HTTP-RPC server listening port (default: 0)
+--rpcmods value    API modules to enable via HTTP-RPC, comma separated list
+--ws               Enable the websockets server
+--wsport value     Websockets server listening port (default: 0)
+--help, -h         show help
+--version, -v      print the version
+```
+
+### Accepted Formats
+
+```
+gossamer [--global-flags] [--local-flags]
+```
+
+```
+gossamer [--local-flags] [--global-flags] 
+```
+
+## Gossamer Subcommands
+
+List of available ***subcommands***:
+
+```
+SUBCOMMANDS:
+    help, h     Shows a list of commands or help for one command
+    account     Create and manage node keystore accounts
+    export      Export configuration values to TOML configuration file
+    init        Initialize node databases and load genesis data to state
+```
+
+List of ***local flags*** for `init` subcommand:
+
+```
+--force            Disable all confirm prompts (the same as answering "Y" to all)
+--genesis value    Path to genesis JSON file
+--verbosity value  Supports levels crit (silent) to trce (trace) (default: "info")
+--name value       Node implementation name
+--chain value       Node implementation id used to load default node configuration
+--config value     TOML configuration file
+--datadir value    Data directory for the node
```

Comment:
is this --basedir now? also --verbosity, --config, --datadir are global not subcommand specific

Reply:
yes, see #868